#!/usr/bin/env python

from dynamic_reconfigure.parameter_generator_catkin import *

PACKAGE_NAME = "squirrel_2d_localizer"

gen = ParameterGenerator()

gen.add("noise_xx", double_t, 0, "", 1.0, 0.0, 10.0)
gen.add("noise_xy", double_t, 0, "", 0.0, 0.0, 10.0)
gen.add("noise_xa", double_t, 0, "", 0.0, 0.0, 10.0)
gen.add("noise_yy", double_t, 0, "", 1.0, 0.0, 10.0)
gen.add("noise_ya", double_t, 0, "", 0.0, 0.0, 10.0)
gen.add("noise_aa", double_t, 0, "", 1.5, 0.0, 10.0)
gen.add("noise_magnitude", double_t, 0, "", 0.5, 0.0, 10.0)

exit(gen.generate(PACKAGE_NAME, "squirrel_2d_localizer", "MotionModel"))